tests/openapi/test_shard_snapshot.py

```
@@ -212,15 +212,15 @@ def test_shard_snapshot_security(collection_name):
     snapshot_name = "/etc/passwd"
     response = requests.get(
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)